README.md

```diff
@@ -159,18 +171,18 @@ class ExampleClassTests: XCTestCase {
 
     let mock = MockExampleClass()
 
-    func testUpdateWasNotCalled() {
+    func testFooWasNotCalled() {
         XCTAssertFalse(mock.invoked(function: "foo"))
     }
 
-    func testUpdateCalledWithBlankByDefault() {
+    func testFooCalledWithBlankByDefault() {
```

Comment:
I don't remember what `BlankByDefault` means, but it should probably be something like `testFooWasCalled`.